packages/terra-form-select/src/native-select/NativeSelect.jsx

```diff
@@ -0,0 +1,244 @@
+import React, {
+  useState,
+  useRef,
+} from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
+import { injectIntl, intlShape } from 'react-intl';
+import ThemeContext from 'terra-theme-context';
+import {
+  defaultPlaceholderValue,
+  isCurrentPlaceholder,
+  getDisplay,
+  getFirstValue,
+  getOptGroupKey,
+  geOptGroupKeyIndex,
+} from './NativeUtils';
+import {
+  optionPropType,
+  optGroupPropType,
+} from './NativePropTypes';
+
+import styles from './NativeSelect.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
```

Comment:
Should we add a first class prop for max-width? We apply it to all the form select examples.

cc @StephenEsser

Reply:
I suppose that would make it a prop not a class.... maybe nevermind here, I wish there was a better way to do this.

Reply:
The field wrappers have a `maxWidth` property consumers can utilize.  I don't think I'd vote for adding the additional prop here unless we plan to add it to all the other variants as well